rollup/default.js

```diff
@@ -0,0 +1,91 @@
+import babel from '@rollup/plugin-babel';
+import commonjs from '@rollup/plugin-commonjs';
+import path from 'path';
+import { nodeResolve } from '@rollup/plugin-node-resolve';
+import { terser } from 'rollup-plugin-terser';
+
+export default [
+	{
+		input: 'src/index.js',
+		output: [
+			{ file: 'dist/index.cjs', format: 'cjs', sourcemap: true, exports: 'auto' },
+			{ file: 'dist/index.mjs', format: 'esm', sourcemap: true, exports: 'auto' },
+		],
+		onwarn: (warning) => {
+			// Silence circular dependency warning for postcss-values-parsers package
+			if (
+				warning.code === 'CIRCULAR_DEPENDENCY' &&
+				warning.importer.indexOf('node_modules/postcss-values-parser/lib') > -1
+			) {
+				return;
+			}
+
+			console.warn(`(!) ${warning.message}`);
+		},
+		plugins: [
+			babel({
+				babelHelpers: 'bundled',
+				exclude: 'node_modules/**',
+				presets: [
+					['@babel/preset-env', {
+						corejs: 3,
+						loose: true,
+						modules: false,
+						targets: { node: 12 },
```

Comment:
Are targets correct here? The current LTS is 16

Reply:
I think so, but now starting to doubt it.
I thought we declare node 12 here as that is the lowest version we actively support.
Otherwise the build output might not work in node 12|14

https://babeljs.io/docs/en/babel-preset-env#targetsnode

Reply:
Makes sense